src/raftstore/store/fsm/peer.rs

```diff
@@ -1315,6 +1320,19 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         Ok(None)
     }
 
+    // Check if this peer can handle request_snapshot.
```

Comment:
From the comments and function name i don't quiet understand it.
I suggest just inline these code in `on_raft_message`.